Sources/Signal.swift

```diff
@@ -549,6 +549,25 @@ extension SignalProtocol {
 		}
 	}
 
+	/// Maps each value in the signal to a new value, lazily evaluating the
+	/// supplied transformation on the specified scheduler.
+	///
+	/// - note: The difference between `lazyMap` and `map` is that `lazyMap`
+	///         will not force the transformation to get invoked immediately
+	///			when a new value is sent. Furthermore, subsequent `.value`
+	///			events will not cause `transform` to repeatedly evaluate when
+	///			it has not been scheduled for execution by `scheduler`.
+	///
```

Comment:
Maybe add an _important_ block that points out that there's not a 1-to-1 mapping between inputs and outputs?